invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/collections/macros.html

```diff
@@ -1,22 +1,22 @@
 {% macro render_depth_one_collection(community, tree, collection) %}
 {% set root_collection=collection[collection.root] %}
-<div class="ui card borderless facet mt-0 mb-0 rel-ml-1">
+<div class="ui mt-0 mb-0 rel-ml-1">
   <div class="content rel-mb-1">
     {% set logo_url = url_for("static", filename="images/collections/" ~ root_collection.slug ~ ".jpg") %}
     <div class="ui grid">
-      <div class="four wide column collection-logo mr-0 computer only">
+      <div class="three wide column collection-logo mr-0 computer only">
         <img src="{{ logo_url }}" width="45" height="auto" alt=""
           onerror="null;"/>
       </div>
-      <div class="ten wide column collection-header middle aligned left floated pl-0">
+      <div class="nine wide column collection-header middle aligned right floated pl-0">
         <h4 class="theme-primary-text">
           <a
             href='{{ url_for("invenio_app_rdm_communities.community_collection", pid_value=community.slug, collection_slug=root_collection.slug, tree_slug=tree.slug )}}'>
             {{ root_collection.title }}
           </a>
         </h4>
       </div>
-      <div class="two wide column collection-number middle aligned">
+      <div class="four wide column collection-number middle aligned left floated">
```

Comment:
The browse page looks much better with these changes, but there is one specific viewport that doesn't look great.
Checking for a solution.

Reply:
Should be solved by my commit.